src/pages/editor/ChannelClaimTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { ClaimedChannel, SelectedPublicationChannelType } from '../../types/customerInstitution.types';
+import { PublicationChannelType } from '../../types/registration.types';
+import { ChannelClaimTableRow } from './PublicationChannelTableRow';
+
+interface ChannelClaimTableProps {
+  channelClaimList: ClaimedChannel[];
+  channelType: SelectedPublicationChannelType;
+}
+
+export const ChannelClaimTable = ({ channelClaimList, channelType }: ChannelClaimTableProps) => {
+  const { t } = useTranslation();
+  const claimedChannels = filterClaimedChannels({ channelClaimList, channelType }) ?? [];
+
+  return (
+    <TableContainer>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>{t('common.publisher')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.channel_owner')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.publishing_access')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.editing_access')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.category_limitations')}</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {claimedChannels.map((channel, index) => (
+            <ChannelClaimTableRow channel={channel} key={index} />
```

Comment:
Kanskje bruke `id` da den ser ut til å være unik her? Ellers vil det bli trøbbel her når det blir støtte for å fjerne/endre claims, da claims vil endre `index` når et element over forsvinner
```suggestion
          {claimedChannels.map((channel) => (
            <ChannelClaimTableRow channel={channel} key={channel.id} />
```